src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/PublicAPI.Unshipped.txt

```diff
@@ -6,4 +6,5 @@ static Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.PropertyPageResourc
 static Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.PropertyPageResources.DeleteBtnHelpText.get -> string
 static Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.PropertyPageResources.NewBtnHelpText.get -> string
 static Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.PropertyPageResources.OKBtnHelpText.get -> string
-static Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.PropertyPageResources.chkLaunchBrowserHelpText.get -> string
\ No newline at end of file
+static Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.PropertyPageResources.chkLaunchBrowserHelpText.get -> string
+static Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.PropertyPageResources.chkNativeCodeDebuggingText.get -> string
```

Comment:
Not specific to this PR but it seems weird to have the UI elements as public fields.

Reply:
Agreed, was looking at this earlier - why are these public API?

Reply:
Never mind. Missed that this was a resource.

Reply:
But why is this public?

Reply:
Not sure, but it is generated as Public when you edit the resx

Reply:
Is making these internal out of the question?

Reply:
I don't believe it is out of the question - I will make a PR for this and we can discuss there